aspnetcore/performance/caching/middleware/samples/2.x/ResponseCachingSample.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFrameworks>net462;netcoreapp2.0</TargetFrameworks>
+    <OutputType>Exe</OutputType>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore" Version="2.0.0" />
```

Comment:
Curious why we're not using `Microsoft.AspNetCore.All` since that's what is used in the default templates?